apps/src/templates/sectionAssessments/FreeResponsesAssessments.jsx

```diff
@@ -0,0 +1,123 @@
+import React, {Component, PropTypes} from 'react';
+import {Table, sort} from 'reactabular';
+import {tableLayoutStyles, sortableOptions} from "../tables/tableConstants";
+import i18n from '@cdo/locale';
+import wrappedSortable from '../tables/wrapped_sortable';
+import orderBy from 'lodash/orderBy';
+
+export const COLUMNS = {
+  STUDENT: 0,
+  RESPONSE: 1,
+};
+
+const freeResponsesDataPropType = PropTypes.shape({
+  id:  PropTypes.number.isRequired,
+  studentId: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  response: PropTypes.string.isRequired,
+});
+
+class FreeResponsesAssessments extends Component {
+  static propTypes= {
+    freeResponses: PropTypes.arrayOf(freeResponsesDataPropType),
+    // responses: PropTypes.arrayOf(textResponsePropType),
+  };
+
+  state = {
+    [COLUMNS.NAME]: {
+      direction: 'desc',
+      position: 0
+    }
+  };
+
+  getSortingColumns = () => {
+    return this.state.sortingColumns || {};
+  };
+
+  onSort = (selectedColumn) => {
+    this.setState({
+      sortingColumns: sort.byColumn({
+        sortingColumns: this.state.sortingColumns,
+        sortingOrder: {
+          FIRST: 'asc',
+          asc: 'desc',
+          desc: 'asc',
+        },
+        selectedColumn
+      })
+    });
+  };
+
+  studentResponseColumnFormatter = (response, {rowIndex}) => {
```

Comment:
I believe since you're just displaying row data in a div, if you named your column properties, `response` and `name` instead of `studentName` and `studentResponse`, and removed the formatters entirely, it would display the same. Reacttabular tries to default to the most common thing if all the names match up.

Even if you do need something custom, I think it makes sense to update the naming. Then `response` will equal `studentResponse` and you won't have to do any lookups, you'll just need to use the parameters. Same in your other formatter.